### ISSUE TITLE
Freezes and crashes on login

### ISSUE THREAD

**Issue by yonyonson** [gittyapp/feedback#31]
I don´t manage to login to my GitHub account. I can start application, choose login with GitHub.com and I can see the front page of mobile version of github.com. However, application freezes when I click either login og password field. It dies afterwards.
I can click other buttons in the interface, but entering username and passord freezes app.

I am using iOS 9.0.2 with iPhone 6s, installed keyboards are Norwegian, Russian and English.
I have just had original Gitty for a while (it was installed on a phone before I have bought Gitty 2).

Screenshot of a frozen app:
![img_0024](https://cloud.githubusercontent.com/assets/315696/10580143/6226e86e-767c-11e5-8fc2-7b1bad71e73d.PNG)

**Comment by lgomez** [comment]
+1. Just paid for it but can't use it.

**Comment by lgomez** [comment]
This was reported a month ago and nothing?

**Comment by yonyonson** [comment]
Nope, I have just installed another app. That was wasted money for me so far.

**Comment by lgomez** [comment]
I requested a refund already and installed CodeHub. Not great but there's nothing out there.

**Comment by jamby** [comment]
:+1: Need this to look at Travis errors.
